vm-setup/roles/v1aX_integration_test/templates/Metal3MachineTemplate.yml

```diff
@@ -9,6 +9,7 @@ metadata:
     name: {{ CLUSTER_NAME }}
     uid: {{ CLUSTER_UID }}
 spec:
+  nodeReuse: true
```

Comment:
Could we parameterize this and set value as "true" on  node_reuse.sh? We could have NODE_REUSE_VAL set to false by default?

Reply:
Good point, will do that in separate PR. Now my priority is to fix the node reuse test and then will refactor other things.